nerfstudio/data/utils/data_utils.py

```diff
@@ -51,3 +54,17 @@ def get_semantics_and_mask_tensors_from_path(
     semantics = torch.from_numpy(np.array(pil_image, dtype="int64"))[..., None]
     mask = torch.sum(semantics == mask_indices, dim=-1, keepdim=True) == 0
     return semantics, mask
+
+
+def get_depth_image_from_path(
+    filepath: Path,
+    height: int,
+    width: int,
+    scene_scale_factor: float,
+    interpolation: int = cv2.INTER_NEAREST,
+) -> torch.Tensor:
+    """Loads, rescales and resizes depth images. Assumes Record3D depth format."""
+    image = cv2.imread(str(filepath.absolute()), cv2.IMREAD_ANYDEPTH)
+    image = image.astype(np.float64) * _MILLIMETER_TO_METER_SCALE_FACTOR * scene_scale_factor
```

Comment:
Should this scale factor be a variable? If not, it should be well documented.

Reply:
Good point, I made it a variable instead in the `NerfstudioDataParserConfig`.